code/go/0chain.net/smartcontract/dbs/event/types.go

```diff
@@ -1,4 +1,4 @@
-package dbs
+package event
```

Comment:
Why are we moving the DbUpdate type into the evnet package?


 The idea of keeping it in the dbs package is to avoid cyclic dependencies, as the type is also used in [storagesc](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/storagesc/eventdb.go#L48)

The dbs package should not import any other 0chain pacakges so safe to include anywhere.

Reply:
https://0chain.slack.com/archives/C02QDMYFQJG/p1648229723268449?thread_ts=1647819416.329659&cid=C02QDMYFQJG

we have an issue we are importing higher package in the lower package this should not be done
we are importing [0chain.net/smartcontract/dbs](http://0chain.net/smartcontract/dbs) in [0chain.net/smartcontract/dbs/event](http://0chain.net/smartcontract/dbs/event)
This is not a good practice